app/controllers/activities_controller.rb

```diff
@@ -4,6 +4,10 @@
 class ActivitiesController < AuthenticatedController
   include ProjectScoped
 
+  def index
+    @activities = Activity.all_latest
```

Comment:
This could be far to many activities. When imports from services happen and they end up created 100,000 issues, there's also 100,000 activities.

I think the approach needs to go back to the drawing board. Inifinite scroll doesn't mean we load everything up front and scroll it, it means we load the next "set" of records and append them to the list, once scrolled we make a new query, and append ~20 more items. This could have performance implications I'm worried about. 

I suppose no more than the `Issues#index`  would 🤔 /cc @aapomm